packages/frontend/.browserslistrc

```diff
@@ -1,3 +1 @@
-> 1%
-last 2 versions
-not dead
+since 2019
```

Comment:
After discussion w/ dim, moving all packages to rely on a "since 2019" browserslist query